ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -10,17 +10,40 @@
+{%             if port_channel_interfaces is arista.avd.defined %}
+{%                 for port_channel_interface in port_channel_interfaces | arista.avd.convert_dicts('name') %}
+{%                     if port_channel_interface_name == port_channel_interface.name  %}
+{%                         if port_channel_interface.type is not defined or port_channel_interface.type not in ['routed', 'l3dot1q', 'l2dot1q'] %}
+{%                             set description = ethernet_interface.description | default("-") %}
+{%                             set mode = port_channel_interface.mode | default("access") %}
+{%                             set vlans = port_channel_interface.vlans | default ('-') %}
+{%                             set native_vlan = port_channel_interface.native_vlan | default ('-') %}
+{%                             set channel_group = ethernet_interface.channel_group.id %}
+{%                             if port_channel_interface.trunk_groups is arista.avd.defined %}
+{%                                 set l2 = namespace() %}
+{%                                 set l2.trunk_groups = [] %}
+{%                                 for trunk_group in port_channel_interface.trunk_groups | arista.avd.natural_sort %}
+{%                                     do l2.trunk_groups.append(trunk_group) %}
+{%                                 endfor %}
+{%                             else %}
+{%                                 set l2 = namespace() %}
+{%                                 set l2.trunk_groups = "-" %}
+{%                             endif %}
+| {{ ethernet_interface.name }} | {{ description }} | *{{ mode }} | *{{ vlans }} | *{{ native_vlan }} | *{{ l2.trunk_groups }} | {{ channel_group }} |
+{%                         endif %}
+{%                     endif %}
+{%                 endfor %}
+{%             else %}
+{%                 set description = ethernet_interface.description | default("-") %}
+{%                 set mode = port_channel_interface.mode | default("access") %}
+{%                 set vlans = port_channel_interface.vlans | default ('-') %}
+{%                 set native_vlan = port_channel_interface.native_vlan | default ('-') %}
 {%                 set channel_group = ethernet_interface.channel_group.id %}
-{%                 if port_channel_interfaces[port_channel_interface].trunk_groups is arista.avd.defined %}
+{%                 if port_channel_interface.trunk_groups is arista.avd.defined %}
 {%                     set l2 = namespace() %}
 {%                     set l2.trunk_groups = [] %}
-{%                     for trunk_group in port_channel_interfaces[port_channel_interface].trunk_groups | arista.avd.natural_sort %}
+{%                     for trunk_group in port_channel_interface.trunk_groups | arista.avd.natural_sort %}
 {%                         do l2.trunk_groups.append(trunk_group) %}
 {%                     endfor %}
 {%                 else %}
```

Comment:
Actually, you can skip everything from line 38-53 but place just 1 line after the line 37 else as below:

(The below one shouldn't be exact, you can your own vars as well.)

```
| {{ ethernet_interface.name }} | {{ ethernet_interface.description | arista.avd.default("-") }} | *access | *- | *- | *- | {{ ethernet_interface.channel_group.id }} |
```